charts/quickwit/templates/control-plane-deployment.yaml

```
@@ -12,7 +13,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
     {{- end }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.control_plane.replicaCount }}
```

```suggestion
  replicas: 1
```
It should always be 1, so I'd rather remove the ability to modify the replica count for the control plane.

---

done